hw/top_earlgrey/ip/ast/ast.core

```diff
@@ -107,7 +107,6 @@ targets:
     default_tool: verilator
     parameters:
       - SYNTHESIS=true
-      - AST_BYPASS_CLK=true
```

Comment:
hmm...i thought this one was needed because of the CDC run time, has that already been solved? 
or should we temporarily create a new core file for cdc?

Reply:
this should not affect CDC since this is a block-level target (there were some lint mismatches due to this being set).
CDC is run at the top-level, and we currently have the `AST_BYPASS_CLK` define set in the CDC run script here:
https://github.com/lowRISC/opentitan/blob/5053c41fc20bc4656a2fdc029a9d16f97f56040a/hw/cdc/tools/verixcdc/run-cdc.tcl#L66-L70

Reply:
ah okay that sounds good. I thought we were doing that through the core files, that sounds good.